datacenterlight/views.py

```diff
@@ -416,6 +419,7 @@ class OrderConfirmationView(DetailView):
     context_object_name = "order"
     model = HostingOrder
 
+
```

Comment:
flake8 is complaining about too many blank lines.

./datacenterlight/templatetags/custom_tags.py:24:1: E302 expected 2 blank lines, found 1